_quickstarts/deploy.markdown

```diff
@@ -0,0 +1,114 @@
+
+#### Apache Kafka®
+
+You will also need a running Apache Kafka® cluster to point Kroxylicious at. The Strimzi project has an [easy-to-follow quickstart](https://strimzi.io/quickstarts/) for setting up a cluster on your local machine using Minikube, or alternatively you can follow the official Apache Kafka® [quickstart](https://kafka.apache.org/documentation/#quickstart) to set up a local bare metal cluster.
+
+Once your cluster is set up, the cluster bootstrap address used by Kroxylicious can be changed in the configuration YAML file (see the [**Configure**](#configure) section below).
+
+<br />
+
+### Downloading Kroxylicious
+
+Kroxylicious can be downloaded from the [releases](https://github.com/kroxylicious/kroxylicious/releases) page of the Kroxylicious GitHub repository, or from Maven Central.
+
+In GitHub, all releases since v0.4.0 have an attached `kroxylicious-app-*-bin.zip` file. Download the latest version of this zip, and optionally verify the contents of the package with the attached `kroxylicious-app-*-bin.zip.asc` file.
+
+{% capture os_archive_note %}
+If you're trying Kroxylicious out on Linux or macOS, you may find the `.tar.gz` format easier to work with. We're using the `.zip` files in this quickstart for cross-platform compatibility, but we recommend you use whichever format you're most familiar with.
+{% endcapture %}
+
+{% include bs-alert.html type="primary" icon="info-circle-fill" content=os_archive_note %}
+
+<br />
+
+# Install
+
+Extract the downloaded Kroxylicious Zip file into the directory you would like to install Kroxylicious in.
+Ensure the `kroxylicious-start.sh` and `run-java.sh` files in the `bin/` directory within the extracted folder have at least read and execute (`r-x`) permissions for the owner.
+
+<br />
+
+# Configure
+
+Kroxylicious is configured with YAML. An example configuration file can be found in the `config/` directory of the extracted Kroxylicious folder, which you can either modify or use as reference for creating your own configuration file.
+
+From the configuration file you can specify how Kroxylicious presents each Apache Kafka® broker to clients, where Kroxylicious will locate the Apache Kafka® cluster(s) to be proxied, and which filters Kroxylicious should use along with any configuration for those filters.
+
+More information about configuring Kroxylicious can be found in the [documentation](https://kroxylicious.io/kroxylicious/#_deploying_proxies).
+
+<br />
+
+# Run
+
+From within the extracted Kroxylicious folder, run the following command:
+
+```shell
+./bin/kroxylicious-start.sh --config config/example-proxy-config.yml
+```
+
+To use your own configuration file instead of the example, just replace the file path after `--config`.
+
+<br />
+
+# Use
+
+To use your Kroxylicious proxy, your client(s) will need to point to the proxy (using the configured Port-Per-Broker or SNI address) rather than directly at the Apache Kafka® cluster.
```

Comment:
SNI can't work without TLS on Kroxy's downstream side.  The feature relies on the TLS client hello. So either the quick start would need to go into setting up TLS and provide example config, we just use port-per-broker for the quickstart.  I'd advocate the latter.

Reply:
Agreed, it would be easier to point people at Port-Per-Broker initially.